frontend/src/products.tsx

```diff
@@ -223,7 +223,7 @@ export const productUrls = {
     replayPlaylist: (id: string): string => `/replay/playlists/${id}`,
     replaySingle: (id: string): string => `/replay/${id}`,
     replayFilePlayback: (): string => '/replay/file-playback',
-    replaySettings: (sectionId?: string): string => `/replay/settings${sectionId ? `?sectionId=${sectionId}` : ''}`,
```

Comment:
Interesting? My hook picked this up I guess...

Reply:
removed this jic